### PR TITLE
BITONBO 444: Generalize KYC error

### PR DESCRIPTION
**Pull request by seguame**
## Associated task:
[BITONBO-444](https://bitsomx.atlassian.net/browse/BITONBO-444)

### Changes: 
This error code can be used with other `kyc` fields other than just `CURP`

### CODE REVIEW

**Review by clausia — APPROVED** [review]

**Review by chochos — APPROVED** [review]

**Review by lrodriguezds — APPROVED** [review]

File: source/index.html.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ error categories, the last two digits define specific errors.
 * 0345: Quote is expired or invalid
 * 0346: You need to agree to the new terms and condition
 * 0347: Too many attempts to validate CURP
-* 0348: CURP data did not match with user data
+* 0348: KYC data did not match with user data
```
Comment (domix) [inline source/index.html.md:178]: while this is correct, but not sure if the API users know what is KYC. 
```suggestion
* 0348: KYC data did not match with user data. (CURP-México, X-Argentina, X-Brasil)
```

makes sense?

**Review by domix — COMMENTED** [review]
a question for now

File: source/index.html.md (hunk 1 above)
Comment (seguame) [reply source/index.html.md]: Agreed, change is now in place.

**Review by domix — APPROVED** [review]

**Review by viccomx — APPROVED** [review]
LGTM

**Pull request merged**